loader/db.go

```diff
@@ -67,14 +75,20 @@ func (conn *Conn) querySQL(ctx *tcontext.Context, query string, args ...interfac
 			if err == nil {
 				cost := time.Since(startTime)
 				queryHistogram.WithLabelValues(conn.cfg.Name).Observe(cost.Seconds())
-				if cost > 1 {
-					ctx.L().Warn("query statement", zap.String("query", query), zap.Reflect("argument", args), zap.Duration("cost time", cost))
+				if cost.Seconds() > 1 {
+					ctx.L().Warn("query statement",
+						zap.String("query", utils.TruncateString(query, stringLenLimit)),
+						zap.Reflect("argument", utils.TruncateInterface(args, stringLenLimit)),
```

Comment:
seems we can use zap.String

Reply:
done in https://github.com/pingcap/dm/pull/257/commits/02aa7c3d02ce604be1d3f8ec03e59aefbe60d7c2.